### ISSUE TITLE
Swagger API 

### ISSUE THREAD

**Issue by marcoherbst** [EyesOfThings/Mobile_Camera_Demonstrator#30]
... if you can.

**Comment by ijunaid8989** [comment]
I have added http://eot.evercam.io/v1/swagger v1 with the only one `/v1/devices` endpoint.

Here were the few hurdles which come across, 

There is only one gem which seems to work perfectly for rails swagger https://github.com/richhollis/swagger-docs, which creates docs for swagger 1.2, and the UI for swagger which load swagger docs doesn't support the swagger 1.2 anymore, it supports swagger 2.0.

So I found another tool to convert swagger docs from one version to another. https://github.com/LucyBot-Inc/api-spec-converter, Now here are the steps to follow to create swagger docs.

1: `SD_LOG_LEVEL=1 rake swagger:docs`
2: `api-spec-converter http://localhost:3000/api-docs.json --from=swagger_1 --to=swagger_2 > swagger.json`
